cdap-api/src/main/java/co/cask/cdap/api/RuntimeContext.java

```diff
@@ -39,4 +40,9 @@
    * @return The application namespace
    */
   String getNamespace();
+
```

Comment:
Are we returning Twill Runids or the ones that is generated in CDAP. Maybe we should just expose the one that is generated in CDAP?